src/app/store/effects/VehicleTechRecordModel.effects.ts

```diff
@@ -180,8 +180,7 @@ export class VehicleTechRecordModelEffects {
                * error not working properly yet
                */
               const errorMessage = error[0].message;
-              this._store.dispatch(new SetErrorMessage([errorMessage]));
-              return [];
+              return [new SetErrorMessage([errorMessage]), new SetViewState(VIEW_STATE.VIEW_ONLY)];
```

Comment:
why do we need to dispatch SetViewState(VIEW_STATE.VIEW_ONLY)?

Reply:
The bug is detailed in the ticket. If the backend PUT returned an error and the user then clicked CANCEL the state was not updated to view.

Reply:
All you need to do here is `return [new SetErrorMessage([errorMessage]);`. The CVSB-14184 bug refers to the `Yes` or `No` of the ADR details not working. That bug in itself is not reproducible because the implementation is already in the VTM-App. Except I am missing something else.  

To clear summary errors when cancel link is clicked please create an effect `clearErrorMessage$` in the error.effects.ts. The effect should have `ofType<SetViewState>` then filter where `VIEW_STATE===VIEW_ONLY`. The final step should be `map(() => ClearErrorMessage()`. This will layout the ground work for error-handling in the VTM-APP at large. More to come when validation ticket is played

Reply:
Let's have a call tomorrow with Dragos and discuss why you don't consider it as bug and why he does. Once clarified, I can make whichever amend is necessary.